lib/client/api_login_test.go

```diff
@@ -77,9 +77,17 @@ func TestTeleportClient_Login_local(t *testing.T) {
 
 	// Reset functions after tests.
 	oldStdin, oldWebauthn := prompt.Stdin(), *client.PromptWebauthn
+	oldHasPlatformSupport := wancli.HasPlatformSupport
+	t.Cleanup(func() {
```

Comment:
I'm a bit confused, is this supposed to be wrapped in a t.Cleanup?

Reply:
no, I guess it was some copy paste mistake :(